tasks/github_tasks.py

```diff
@@ -188,3 +191,79 @@ def get_token_from_app(_, app_id_env='GITHUB_APP_ID', pkey_env='GITHUB_KEY_B64')
     from .libs.common.github_api import GithubAPI
 
     GithubAPI.get_token_from_app(app_id_env, pkey_env)
+
+
+def _get_teams(changed_files, owners_file='.github/CODEOWNERS') -> List[str]:
+    codeowners = read_owners(owners_file)
+
+    team_counter = Counter()
+    for file in changed_files:
+        owners = [name for (kind, name) in codeowners.of(file) if kind == 'TEAM']
+        team_counter.update(owners)
+
+    team_count = team_counter.most_common()
+    if team_count == []:
+        return []
+
+    _, best_count = team_count[0]
+    best_teams = [team for (team, count) in team_count if count == best_count]
+    best_teams = [team.casefold() for team in best_teams]
```

Comment:
```suggestion
    best_teams = [team.casefold() for (team, count) in team_count if count == best_count]
```
Can't you do both in one call?